docs/developing.asciidoc

```diff
@@ -0,0 +1,80 @@
+// begin header
+// end header
+
+toc::[]
+
+== Setting up Certman-operator for local dev
+=== Requirements
+To develop certman you will need the following:
```

Comment:
A bit of semantics, but I believe this should be referred to as `certman-operator` as `certman` is a different tool.